### ISSUE TITLE
cannot call host with long running repl command

### ISSUE THREAD

**Issue by bmoore-msft** [microsoft/poshtools#451]
If I enter a long running command in the REPL window and then try to instantiate the host, by running a script in VS via AzureRM - I get a error "There is a problem in the PowerShell host service."

Honestly not sure what to expect here - probably another instance of the host/repl?

**Comment by ericdevg** [comment]
Before we having multiple host/repl supported(I believe there was a task for that?), we should at least gracefully exit from the latter execution and give message to user saying something else is running, please wait till current one exits.

**Comment by bmoore-msft** [comment]
Yeah we had the task to spin up multiple in VS as having the separate runspace will be useful at times...  This feels like one of those - esp when we have more experiences in VS using PS to accomplish things we used to do in C#.

I think that's the right case here... but need some thought around having new hosts for each caller or having the caller manage the host and REPL associated with it.  I don't think this is a bug, probably more of a feature/design we need to think through.

**Comment by ASayre** [comment]
For now, we should at least make the error message friendlier while in this case, until we figure out multi-host scenarios.
